server/bin/unittests

```diff
@@ -324,7 +332,7 @@ function _setup_state {
     echo "$_testopt/bin/pbench-server-config-activate ${_testtmp}/pbench-server.cfg" >> $_testactout
     $_testopt/bin/pbench-server-config-activate ${_testtmp}/pbench-server.cfg >> $_testactout
     rc=$?
-    rm ${_testtmp}/pbench-server.cfg
+    rm ${_testtmp}/*pbench-server.cfg
```

Comment:
If we move it to the default location, we don't need this removal here.  And, in fact, doesn't removing it here mean that during test runs tests won't have the config file in place?

Reply:
No - once the (top-level) config file is assembled from its various pieces, `pbench-server-config-activate` copies it *and all other config files it finds in the same directory* to the destination directory, so the config file in the tmp directory can safely be deleted.